src/plugins/foxPage/components/Layout.stories.tsx

```diff
@@ -101,11 +108,17 @@ export const FoxLayout: Story = () => {
                 <Text>{'Foxy Page'}</Text>
               </Stack>
               <Stack flex='1 1 0%' width='full' maxWidth={{ base: 'full', xl: 'sm' }} spacing={4}>
-                <Text>
-                  {
-                    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque malesuada rutrum erat. Aliquam commodo tincidunt ligula, sollicitudin semper velit aliquet id. Vestibulum at tincidunt diam. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam facilisis nisi vitae risus vulputate bibendum. Etiam vitae leo ac mauris congue varius. Nunc sit amet diam bibendum, hendrerit velit vel, porta sapien. Aenean ac enim ornare, consectetur justo at, sodales massa. Fusce sodales sapien ac dictum sagittis. Maecenas ornare ex at dolor tempus, ac iaculis dolor dapibus.'
-                  }
-                </Text>
+                <AssetActions
+                  assetIcon={mockAsset.icon}
+                  assetSymbol={mockAsset.symbol}
+                  description={mockAsset.description}
+                  buyCTA={translate('plugins.foxPage.buyAssetOnCoinbase', {
+                    assetSymbol: mockAsset.symbol,
+                  })}
+                  sellCTA='plugins.foxPage.receive'
+                  onReceiveClick={() => null}
+                  onBuyClick={() => null}
+                />
```

Comment:
Same thing regarding Storybook, they might look to be the exact same across the three assets, but since this is just a story, actual usage will be programmatic and there will be no repetition